guides/source/testing.md

```diff
@@ -1431,7 +1430,7 @@ class UserMailerTest < ActionMailer::TestCase
     assert_equal ['me@example.com'], email.from
     assert_equal ['friend@example.com'], email.to
     assert_equal 'You have been invited by me@example.com', email.subject
-    assert_equal read_fixture('invite').join, email.body.to_s
+    assert_equal read_fixture('invite').join, email.text_part.body.to_s
```

Comment:
I do not believe they are both the same. In my testing, for both single and multipart mails email.body.to_s simply returns an empty string.

Might be some version issue going on if you're seeing different. Let me know if that's the case and I'll dig a bit more.

However, apart from that: I think given that the fixture shown is clearly text, we ought to be explicitly testing the text part. And in general I would test the text and HTML parts separately.